src/introduction.md

```diff
@@ -1,42 +1,39 @@
 # Introduction
 
-[Substrate] is a blockchain and smart contract framework written in the Rust programming language.
-The InterPlanetary File System (or [IPFS]) is a distributed, peer-to-peer,
-content-addressed storage framework.
+`offchain::ipfs` is [Substrate], infused with [IPFS].
```

Comment:
```suggestion
`offchain::ipfs` is [Substrate] infused with [IPFS].
```

Reply:
I like the comma :) I'm gonna keep it for now